ai_diffusion/settings.py

```diff
@@ -58,6 +58,12 @@ class ApplyRegionBehavior(Enum):
     no_hide = _("Layer group (don't hide)")
 
 
+class SaveImageFormat(Enum):
```

Comment:
Rather than introducing a new Enum I think it would be better to use the existing `ImageFileFormat`.

You can adapt it to fit the settings UI if needed, eg. could look like this
```python
class ImageFileFormat(Enum):
    png = "PNG (fast)"
    png_small = "PNG"
    webp = "WebP"
    webp_lossless = "WebP (lossless)"
    jpeg = "JPEG"

    @property
    def extension(self):
        if self in [ImageFileFormat.png, ImageFileFormat.png_small]:
            return "png"
        elif self in [ImageFileFormat.webp, ImageFileFormat.webp_lossless]:
            return "webp"
        else:
            return "jpg"

    @property
    def quality(self):
        ...
```